addons/explore-editor-theme/tabs/EditorIcons.gd

```diff
@@ -72,8 +75,16 @@ func _on_icon_item_selected(item_index : int) -> void:
 	
 	icon_preview.texture = icon_texture
 	icon_title.text = icon_name
-	icon_code.code_text = "get_icon(\"" + icon_name + "\", \"" + type_name + "\")"
+	icon_code.code_text = "get_icon('" + icon_name + "', '" + type_name + "')"
```

Comment:
Hmm, why change for single quotes there?

It's not even following the [Godot's guidelines](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#quotes). Though it states that we can use single quotes to avoid escapes here it's important to preserve double quotes as this is a line that is going to be pasted into someone's GDScript file, where it would have no business being single-quoted.

Reply:
good catch. I personally prefer single quotes so I changed that for every code preview in your addon, but I forgot to exclude this onefrom the PR. I excluded the single quotes in the other files as I know that they aren't godot's best practice but missed this one. Will fix

Reply:
Bike-shedder in me wants to make it a setting now 😄

Reply:
I already thought about doing that and creating a separate PR... but not now :D